src/basics.md

```diff
@@ -847,6 +848,31 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-check-cpu-cores]: #ex-check-cpu-cores
+<a name="ex-check-cpu-cores"></a>
+## Check number of logical cpu cores
+
+[![num_cpus-badge]][num_cpus] [![cat-hardware-support-badge]][cat-hardware-support]
+
+Counts the number of logical cpu cores in current machine.
+
+```rust
+# #[macro_use]
+# extern crate error_chain;
+extern crate num_cpus;
+#
+# error_chain! {
```

Comment:
No need for `error_chain` as there is no possibility for error handling in this example. Lets just remove it.